ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -7,6 +7,206 @@
+      regexKo: Regexes.startsUsing({ id: '4BAA', source: '라무', capture: false }),
+      infoText: {
+        en: 'Fury\'s Bolt',
+        ko: '라무 강화',
+      },
+    },
+    {
+      id: 'E5S Judgment Volts',
+      regex: Regexes.startsUsing({ id: '4BB5', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BB5', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BB5', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BB5', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BB5', source: '라무', capture: false }),
+      infoText: {
+        en: 'aoe',
+        de: 'AoE',
+        fr: 'Dégâts de zone',
+        cn: 'AOE',
+        ko: '전체 공격',
+      },
+    },
+    {
+      id: 'E5S Stepped Leader',
+      regex: Regexes.startsUsing({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BC6', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BC6', source: '라무', capture: false }),
+      preRun: function(data) {
+        data.leaderCount = data.leaderCount || 0;
+      },
+      alertText: function(data) {
+        if (data.leaderCount %2 == 0) {
+          return {
+            en: 'Ready Spread',
+            ko: '산개 준비',
+          };
+        } else if (data.leaderCount %2 == 1) {
+          return {
+            en: 'donut AoE',
+            ko: '도넛 장판',
+          };
+        }
+      },
+      run: function(data) {
+        data.leaderCount++;
+      }
+    },
+    {
+      id: 'E5S Stepped Leader Spread',
+      regex: Regexes.ability({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.ability({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.ability({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.ability({ id: '4BC6', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.ability({ id: '4BC6', source: '라무', capture: false }),
+      alarmText: function(data) {
+        if (data.leaderCount %2 == 0) {
+          return {
+            en: 'Spread',
+            de: 'Verteilen',
+            fr: 'Ecartez-vous',
+            ja: '散開',
+            cn: '分散',
+            ko: '산개',
+          };
+        }
+      },
+    },
+    {
+      id: 'E5S Crippling Blow',
+      regex: Regexes.ability({ id: '4BCA', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.ability({ id: '4BCA', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.ability({ id: '4BCA', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.ability({ id: '4BCA', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.ability({ id: '4BCA', source: '라무', capture: false }),
```

Comment:
I think this should not be capture false if you are using matches below.  I'm surprised `npm run test` didn't complain.  (cc @panicstevenson)

Reply:
a ha, capture is that thing. I didn't know.

Reply:
That complained.